bindings/matrix-sdk-ffi/src/widget.rs

```diff
@@ -113,17 +113,16 @@ pub async fn generate_webview_url(
 
 /// Defines if a call is encrypted and which encryption system should be used.
 ///
-/// This controls the url parameters: `enableE2EE`, `perParticipantE2EE`,
-/// `password`.
+/// This controls the url parameters: `perParticipantE2EE`, `password`.
 #[derive(uniffi::Enum, Clone)]
 pub enum EncryptionSystem {
     /// Equivalent to the element call url parameter: `enableE2EE=false`
     Unencrypted,
     /// Equivalent to the element call url parameters:
```

Comment:
(sorry about the nitpick, trying to be consistent with the previous comments :grin:)

```suggestion
    /// Equivalent to the element call url parameter:
```